kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt

```diff
@@ -101,7 +103,8 @@ class GeneratedAdaptersTest {
     }
 
     @Language("JSON") val expected = """{"foo":"fooString","bar":"","bazList":[]}"""
-    assertThat(adapter.toJson(DefaultValues("fooString"))).isEqualTo(expected)
+    assertThat(adapter.toJson(
+        DefaultValues("fooString"))).isEqualTo(expected)
```

Comment:
what’s with the weird wrapping?

Reply:
Probably intellij autoformatting weirdness after the package name change. There's an open issue to use spotless once PRs die down